common/journal_entries/dvg_ameriga.txt

```diff
@@ -0,0 +1,99 @@
+
+
+
+je_ameriga_unite = {
+	icon = "gfx/interface/icons/event_icons/event_military.dds"
+
+	is_shown_when_inactive = {
+
+
+	exists = c:AME
+	
+	this = c:AME
+	
+		
+	}
+
+    possible = {
+	}
+
+	complete = {
+		is_diplomatic_play_committed_participant = no
+		any_scope_state = {
+		
+		count >= 5
+				
+			AND = {	
+				region = sr:region_brazil
+				
+				
+				
+				owner = ROOT
+
+			}
+			}
+		any_scope_state = {
+		
+		count >= 5
+				
+			AND = {	
+				region = sr:region_la_plata
+				
+				
+				owner = ROOT
+
+			}
+			}
+		any_scope_state = {
+		
+			count >= 5
+				
+			AND = {	
+				region = sr:region_gran_colombia
+				
+				
+				owner = ROOT
+	
+			}
+			}
+		any_scope_state = {
+		
+			count >= 5
+				
+			AND = {	
+				region = sr:region_andes
+				
+				
+				owner = ROOT
+	
+			}
+			}
+		
+			}
+    on_complete = {
+		trigger_event = { id = dvg_ameriga.3 popup = yes }
+		
+    }
+	fail = {
+	
+	AND = {
+	
+	any_interest_group = { is_insurrectionary = yes }
+	
+	turmoil > 0.9

```

Comment:
Inconsistent trigger scopes (country vs. state) in file: common/journal_entries/dvg_ameriga.txt line: 83